readthedocs/vcs_support/utils.py

```diff
@@ -37,47 +37,41 @@ def __init__(self, project, version, timeout=5, polling_interval=0.1):
 
     def __enter__(self):
         start = time.time()
+        log.bind(
+            project_slug=self.name,
+        )
         while os.path.exists(self.fpath):
             lock_age = time.time() - os.stat(self.fpath)[stat.ST_MTIME]
             if lock_age > self.timeout:
-                log.debug(
-                    'Lock (%s): Force unlock, old lockfile',
-                    self.name,
-                )
+                log.debug('Force unlock, old lockfile.')
                 os.remove(self.fpath)
                 break
-            log.debug('Lock (%s): Locked, waiting..', self.name)
+            log.debug('Locked, waiting...')
             time.sleep(self.polling_interval)
             timesince = time.time() - start
             if timesince > self.timeout:
-                log.debug(
-                    'Lock (%s): Force unlock, timeout reached',
-                    self.name,
-                )
+                log.debug('Force unlock, timeout reached')
                 os.remove(self.fpath)
                 break
             log.debug(
-                '%s still locked after %.2f seconds; retry for %.2f'
-                ' seconds',
-                self.name,
-                timesince,
-                self.timeout,
+                'Lock still locked after some time. Retrying in some seconds',
+                locked_time=timesince,
+                seconds=self.timeout,
             )
         open(self.fpath, 'w').close()
-        log.debug('Lock (%s): Lock acquired', self.name)
+        log.debug('Lock acquired.')
 
     def __exit__(self, exc, value, tb):
+        log.bind(project_slug=self.name)
```

Comment:
Does this unbind it, since we're also setting this on `__enter__`? Could definitely use a comment.

Reply:
I explained this in https://github.com/readthedocs/readthedocs-corporate/pull/1336#issuecomment-977167702